tests/SafeStream/SafeStream.basic.phpt

```diff
@@ -23,3 +23,40 @@ unlink('nette.safe://myfile.txt');
 
 // this is not thread safe - don't relay on returned value
 Assert::false(is_file('nette.safe://myfile.txt'));
+
+
+/* test for metadata */
+// create file
+Assert::true(touch('nette.safe://test.txt'));
+// is file?
+Assert::true(is_file('nette.safe://test.txt'));
+// default permission
+Assert::equal('100644', sprintf('%o', fileperms('nette.safe://test.txt')));
```

Comment:
This assumption is incorrect, 100644 may not always be the default permission (and that's probably why Travis failed - it defaults to 002, but you expect 022).